pkg/config/config.go

```diff
@@ -93,5 +95,23 @@ func LoadSpecFromFile(configPath string) (*Spec, error) {
 		return nil, wErrors.Wrap(err, "could not unmarshal config file")
 	}
 
-	return cfgSpec, nil
+	// replace matchers aliases in tables with matchers expressions
+	for i, t := range cfgSpec.Tables {
+		if t.Filter.Match == "" {
+			continue
+		}
+
+		if m, ok := cfgSpec.Matchers[t.Filter.Match]; ok {
+			cfgSpec.Tables[i].Filter.Match = m
+			continue
+		}
+
+		// matcher keys can be lower-cased by the parser - check this case as well
+		if m, ok := cfgSpec.Matchers[strings.ToLower(t.Filter.Match)]; ok {
+			cfgSpec.Tables[i].Filter.Match = m
+			continue
+		}
```

Comment:
We probably need a warning/error when the matcher is not found right?
Then again based on https://github.com/hellofresh/klepto/pull/113/files#diff-7a85f97083ec2581345f70e1e9ce4e22L156-L160 we just assume it's a proper expression.....

Based document this some where?

Reply:
Matcher in the table can be either expresson or alias - if it is expression already we leave it as it is, but if it is an alias listed in `Matchers` section - we replace it with the expression. Previously this was done in query generation code, but then it was required to pass aliases map everywhere. Not all the aliases are changed to expressions here.

I'll update readme as it mentions that it always must be an alias while it used to work with both.